js/parts/Options.js

```diff
@@ -1765,6 +1765,84 @@ H.defaultOptions = {
          */
         widthAdjust: -44
     },
+    /**
+     * The chart's caption, which will render below the chart and will be part
+     * of exported charts. The caption can be updated after chart initialization
+     * through the `Chart.update` or `Chart.caption.update` methods.
+     *
+     * @sample highcharts/caption/text/
+     *         A chart with a caption
+     * @since  next
+     */
+    caption: {
```

Comment:
In my opinion `caption.enabled` would be useful to keep it consistent with other features.
I see that this is similar to `title`, but I believe to have the option `enabled` as in `credits` is a nicer pattern because you can just toggle it on an off.

Reply:
In that case we need to change title and subtitle too